crates/bevy_internal/Cargo.toml

```
@@ -77,7 +77,7 @@ serialize = [
 multi-threaded = [
   "bevy_asset/multi-threaded",
```

```suggestion
  "bevy_asset?/multi-threaded",
```
could you also change it for bevy_asset?

---

Done